src/resources/views/editor.blade.php

```diff
@@ -1,12 +1,12 @@
 $(function(){
-    window.LaravelDataTables = window.LaravelDataTables || {};
+    window.{{ config('datatables.variable_name') }} = window.{{ config('datatables.variable_name') }} || {};
```

Comment:
We should add a default value here because this can cause a breaking change if not set. Applies to all config call.

```php
config('datatables.variable_name', 'LaravelDataTables')
```

Reply:
Also config should be under `datatables-html` package.

```php
config('datatables-html.variable_name', 'LaravelDataTables')
```

Reply:
Maybe we can use `namespace` instead of `variable_name`?

```php
config('datatables-html.namespace', 'LaravelDataTables')
```

Reply:
Your suggestions is make sense. I will do.